__test__/index.test.js

```diff
@@ -539,4 +550,93 @@ describe("index.js", () => {
       expect(nextReleaseTag).toBe(`v${todayDate}.3`);
     });
   });
+
+
+  describe ("getClubhouseWorkFlowId", ()=> {
+    test("calls the clubhouse workflow api and returns the Engineering workflow id", () =>{
+      const params = {
+        headers: {
+          "Content-Type": "application/json",
+          "Clubhouse-Token": "some-ch-token"
+        },
+      };
+      const response = {
+        data: [
+          {
+            id: 1111111,
+            name: "Engineering",
+            states:[
+              {
+                description: 'Merged to master and waiting on deployment',
+                name: 'Merged',
+                id: 123456,
+              },
+              {
+                description: 'Stories that have been deployed to production as part of a release',
+                name: 'Deployed',
+                id: 54321            
+              }
+            ]
+          },
+          {
+            id: 2222222,
+            name: "Design",
+            states:[
+              {
+                description: 'Merged to master and waiting on deployment',
+                name: 'Merged',
+                id: 999999,
+              },
+              {
+                description: 'Stories that have been deployed to production as part of a release',
+                name: 'Deployed',
+                id: 888888           
+              }
+            ]
+          }
+        
+        ],
+      };
+
+      axios.get.mockImplementationOnce(() => Promise.resolve(response));
+      const workFlowId = main.getClubhouseWorkFlowId();
+
+      expect(axios.get).toHaveBeenCalledWith(
+        "https://api.clubhouse.io/api/v3/workflows",
+        params
+      );
+
+      return workFlowId.then((id) => {
+        expect(id).toBe(54321);
+      });
+    })
+  });
+
+  describe ("updateMultipleStories", ()=> {
+    test("calls the clubhouse bulk update api ", () =>{
+      const params = {
+        headers: {
+          "Content-Type": "application/json",
+          "Clubhouse-Token": "some-ch-token"
+        },
+      };
+
+      const payload = {
+        story_ids: [8993,6456],
+        workflow_state_id: 123
+      }
+      const response = {
+        data: []
+      };
+
+      axios.put.mockReturnValueOnce(Promise.resolve(response));
+      main.updateMultipleStories(123, "8993,6456");
```

Comment:
Do we use a formatted string as the parameter because it is the way the data is delivered? If not, then it may make more sense to use a more appropriate typing like a list of numbers.

Reply:
I agree! However the list of clubhouse numbers are passed in by calling core.getInput("...."), which I believe (or at least based on my observation and experiment) will always be a string

Reply:
I tried a list of numbers, e.g. `[8993, 6456]`, and when it is retrieved, it ended up as `"[8993, 6456]"`